packages/mask/shared/kv-storage.ts

```diff
@@ -21,6 +21,7 @@ export const InMemoryStorages = {
         schema: 1,
         chainId: 1,
         pluginId: '',
+        nftUsage: 0,
```

Comment:
I guess this should not be related to `WalletStateBar`

Reply:
removed